content/development/mining-resources/pools.en.yaml

```diff
@@ -171,6 +171,10 @@
   name: Open ETC Pool
   link: https://github.com/etclabscore/open-etc-pool
   rank: 5
+- 
```

Comment:
key: GreenFox
name: GreenFox
link: https://etc.greenfox.pw

Reply:
@j00keRR please open a PR for GreenFox addition and add it corresponding to the alphabetical order. Thank you